ui/src/project.ts

```diff
@@ -24,7 +24,7 @@ export interface Project {
   id: string;
   shareableEntityIdentifier: string;
   metadata: Metadata;
-  stats: Stats;
+  stats?: Stats;
```

Comment:
Optionality here depends on whether or not we use a different `type` for `Discovery` projects.